tests/test-openshift_provision-generate_resource.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Set Facts
+  hosts: localhost
+  connection: local
+  vars_files:
+  - login-creds.yml
+  tasks:
+  - include_tasks: setup-test.yml
+  - set_fact:
+      ignore_differences:
+        metadata:
+          annotations:
+            kubectl.kubernetes.io/last-applied-configuration: ''
+          creationTimestamp: null
+          selfLink: null
+          namespace: provision-test
+        spec:
+          sessionAffinityConfig:
+            clientIP:
+              timeoutSeconds: 10800
+
+- name: Test Provision
+  hosts: localhost
+  connection: local
+  vars:
+    provision_service: >-
+      {{ lookup('template', 'resources/test-service.yml.j2')
+       | from_yaml }}
+    service_ports:
+    - name: 8080-tcp
+      port: 8080
+
+  roles:
+  - role: openshift-provision
+    openshift_clusters:
+    - projects:
+      - name: provision-test
+
+  tasks:
+  - name: Provision Service
+    openshift_provision:
+      connection: "{{ openshift_connection }}"
+      namespace: provision-test
+      resource: "{{ provision_service }}"
+      generate_resources: True
+
+  - name: Get Service
+    command: cat ./manifests/provision-test_Service_test-service.json
+    register: get_service
+    changed_when: false
+
+  - name: Verify Service
+    fail:
+      msg: |
+        Service not defined as expected
+        >>>
+        {{ cmp_service | to_yaml }}
+        ===
+        {{ got_service | to_yaml }}
+        <<<
```

Comment:
Line 56, 58, and 60 look messed up to me.

Reply:
@seanmalloy I don't believe this is messed up, I believe I had used a different test as an example.
Ref: https://github.com/KohlsTechnology/ansible-role-openshift-provision/blob/6db8c0e505138ebdd4a7d2d5c91861f745d519e2/tests/test-cluster_resources-ClusterResourceQuota.yml#L94-L102

Reply:
@vinny-sabatini sounds good